### PR TITLE
Release Re-Auth and AuthTokenManager

### PR DESCRIPTION
**Pull request by bigmontz**
Removes the @experimental tags form the APIs.
The API is consider stable.

### CODE REVIEW

File: packages/core/src/auth-token-manager.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@ export type SecurityErrorCode = `Neo.ClientError.Security.${string}`
 /**
  * Interface for the piece of software responsible for keeping track of current active {@link AuthToken} across the driver.
  * @interface
- * @experimental Exposed as preview feature.
  * @since 5.8
```
Comment (thelonelyvulpes) [inline packages/core/src/auth-token-manager.ts]:
```suggestion
 * @since 5.14
```
Should this be when it was GA'd?

**Review by thelonelyvulpes — APPROVED** [review]
I just had a question about the use of `@since`, Otherwise, that's a nice clean mechanism in js for marking experimental features, which makes this change very simple.

File: packages/core/src/auth-token-manager.ts (hunk 1 above)
Comment (bigmontz) [reply packages/core/src/auth-token-manager.ts]: That makes sense.

**Comment by bigmontz** [comment]
[Domain Object Mapping [Javascript]](https://trello.com/c/Krl7PBEB/304-domain-object-mapping-javascript)

**Pull request merged**